tcms/testruns/templates/testruns/get.html

```diff
@@ -263,7 +263,7 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                                 class="bug-tooltip"
                                                                 style="visibility: hidden;"
                                                                 data-toggle="popover" data-html="true"
-                                                                data-content="undefined"
+                                                                data-content=""
```

Comment:
Bring this back on. If something fails users will see "undefined" and are more likely to report a bug.